daemon/oci_linux.go

```diff
@@ -463,13 +463,17 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		}
 
 		if m.Source == "tmpfs" {
-			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode}
+			options := "noexec,nosuid,nodev," + volume.DefaultPropagationMode + ",size=65536k"
```

Comment:
I don't understand point of using string here if you're immediately splitting it in `MergeTmpfsOptions`